### PR TITLE
Add issue and pr templates for new copy translations

### LINKED ISSUE #5681: Create template to submit "new copy" translation requests
Once we start supporting multiple languages. Adding new texts/copies throughout the app will need a little bit more of preparation.

Ideally, new copy issues should be opened and closed before the new copies are close to be released.

This template should be something in between #4835 and #4837, but it should contain the list of languages that need translation and should only be closed after all of them are merged into develop (or a project branch).

### LINKED ISSUE #5682: Create a manual PR template (to copy and paste) for "new copy" translations
This PR should be very similar to the PR templated introduced by #4838, except that it will be used for new translations.

### PR DESCRIPTION
**Pull request by daividssilverio**
## What's this?
This adds the missing issue/PR templates for handling new copy translations

### Relationships
Closes #5681
Closes #5682 

## Why do we want this?
To have a way to handle new copies as well.

## How is it done?
The issue and PR templates are added.

### Why not another way?
The described process seems alright, feel free to make suggestions.

## :squid: Permissions
🦑 it

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/i-new-copy-translation-request.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+---
+## Target Languages: <!-- Requested Languages Here -->
+<!-- This kind of issue should usually only be created by the Toggl people, so please make sure you are listing all currently supported languages. -->
+
+## Requested at: <!-- enter date of request -->
+
+## Progress
+<!-- Let's say we have X, Y languages supported -->
+
+- [ ] Language X in progress
```
Comment (juanlaube) [inline .github/ISSUE_TEMPLATE/i-new-copy-translation-request.md]: Maybe put the actual supported languages here?

File: .github/PULL_REQUEST_TEMPLATE/new-copy-translation-pull-request-template.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## New Copy Translation for <!-- Language Here -->
+
+Source request issue: #SourceNewCopyTranslationIssueNumber
```
Comment (dshr) [inline .github/PULL_REQUEST_TEMPLATE/new-copy-translation-pull-request-template.md]: Maybe replace this with comments? Like:
```suggestion
Source request issue: #<!-- insert original issue number -->
```

File: .github/PULL_REQUEST_TEMPLATE/new-copy-translation-pull-request-template.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Request issue: #TranslationIssueNumber
```
Comment (dshr) [inline .github/PULL_REQUEST_TEMPLATE/new-copy-translation-pull-request-template.md]: Same thing:
```suggestion
Request issue: #<-- issue number -->
```

File: .github/ISSUE_TEMPLATE/i-new-copy-translation-request.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+---
+name: "🌏 New Copy Translation Request"
```
Comment (dshr) [inline .github/ISSUE_TEMPLATE/i-new-copy-translation-request.md]: Maybe drop the "copy" part?

Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/i-new-copy-translation-request.md]: But, this is a new copy translation request 🤔

File: .github/ISSUE_TEMPLATE/i-new-copy-translation-request.md (hunk 1 above)
Comment (daividssilverio) [reply .github/ISSUE_TEMPLATE/i-new-copy-translation-request.md]: We are killing this section in favor to Project's columns.

**Pull request merged**